Umbicosaurus/Umbicosaurus.Demo/App_Plugins/Umbicosaurus/thesaurus.json

```diff
@@ -3462,14 +3477,19 @@
       "hub",
       "virus"
     ],
-    "group": "item misc",
+    "group": "item science",
```

Comment:
Thats a fair comment, i will counter i have no problem with that but i think we should also have realistic descriptors because some people will use that as a search term..

extending a metaphor 

say i am building a sci fi website and because I am a nerd 
i want my icons to be science my first search may well be science 
and then choosing the right science based icon..

also if we dont have a direct thesaurus aren't we missing the point..

Reply:
Yes, you're right there.

I definitely think we need to include synonyms for the intended meaning alongside the alternative definitions. But we need to also include synonyms for the real-world use-cases, which I'd argue are very seldom "molecules"!

(I'm regretting more and more using the word "thesaurus" for this! 😂)

As for the category, my viewpoint is that the most common usage should prevail here. The group property is used to display icons in a more logical sort order. In this case, I'd argue the icon sits better with ![icon-network-alt](https://user-images.githubusercontent.com/3120611/142224322-2f363808-a5ed-4eb3-88af-03bebafd8edf.png)
 than ![icon-lab](https://user-images.githubusercontent.com/3120611/142224386-bd114638-1eb9-4012-bd31-d4c04ae8c60f.png).

I do see your point, though. Is there a use case for people searching for "science" and seeing this icon? Or should we reserve adding a thesaurus entry of `science` for icons that could represent science as a whole (e.g. icon-lab)?

Maybe at some point we need a one-to-many relationship with groupings, but currently there's no way for the UI to support this - perhaps this is something for later?

Reply:
I need to have a think..  and see how I might do it..

I think we can do it easily in the data, but the ui part might need to be thought through.. but we are better having more data rather than less and then trimming it if we need

so my follow up using  icon-molecule as an example does the existing "search look at the file name .. because if it does then we can focus on the synonyms  and the categorisation and grouping data

we could think of a taxonomy of 
Science  ( too broad) there for not useful

**> Covers both Icon network alt and icon lab

science  chemistry**

> icon lab

**science computers

> icon network alt**